### DISCUSSION TITLE
Generate scatter plot of two features

### DISCUSSION THREAD

**Discussion by peggiechien** [GreenleafLab/ArchR#815] [Questions / Documentation]
Hello, is there a way to generate a scatter plot between two features using the gene score matrix? I'm thinking something similar to this as done in Seurat: https://satijalab.org/seurat/reference/featurescatter

Thank you!

**Comment by rcorces** [comment]
I imagine one of these is what you are looking for:
https://www.archrproject.com/reference/ggPoint.html
https://www.archrproject.com/reference/ggOneToOne.html
